flang/test/Semantics/typeinfo09.f90

```diff
@@ -0,0 +1,20 @@
+!RUN: bbc --dump-symbols %s | FileCheck %s
+!RUN: %flang_fc1 -fdebug-dump-symbols %s | FileCheck %s
+! test setting of isargcontiguousset in the runtime type info.
```

Comment:
Aren't we checking for `isargdescriptorset`?

Reply:
Absolutely, thanks for reading it!